src/main/java/frc/trigon/robot/subsystems/arm/Arm.java

```diff
@@ -0,0 +1,130 @@
+            followerElevatorMotor = ArmConstants.FOLLOWER_ELEVATOR_MOTOR;
+    private final CANcoder angleEncoder = ArmConstants.ANGLE_ENCODER;
+    private final TalonSRX elevatorEncoder = ArmConstants.ELEVATOR_ENCODER;
+    private TrapezoidProfile
+            angleMotorProfile = null,
+            elevatorMotorProfile = null;
+    private double
+            lastAngleMotorProfileGenerationTime,
+            lastElevatorMotorProfileGenerationTime;
+
+    public static Arm getInstance() {
+        return INSTANCE;
+    }
+
+    private Arm() {
+
+    }
+
+    public Command getSetTargetArmAngleCommand(Rotation2d angle) {
+        return new FunctionalCommand(
+                () -> generateAngleMotorProfile(angle),
+                this::setTargetAngleFromProfile,
+                (interrupted) -> {
+                },
+                () -> false,
+                this
+        );
+    }
+
+    public Command getSetTargetElevatorPositionCommand(double targetElevatorPosition) {
+        return new FunctionalCommand(
+                () -> generateElevatorMotorProfile(targetElevatorPosition),
+                this::setTargetPositionFromProfile,
+                (interrupted) -> {
+                },
+                () -> false,
+                this
+        );
+    }
+
+    private void generateAngleMotorProfile(Rotation2d targetAngle) {
+        angleMotorProfile = new TrapezoidProfile(
+                ArmConstants.ANGLE_CONSTRAINTS,
+                new TrapezoidProfile.State(targetAngle.getDegrees(), 0),
+                new TrapezoidProfile.State(getAngleMotorPositionDegrees().getDegrees(), getAngleMotorVelocityDegreesPerSecond())
+        );
+        lastAngleMotorProfileGenerationTime = Timer.getFPGATimestamp();
+    }
+
+    private void generateElevatorMotorProfile(double targetElevatorPosition) {
+        elevatorMotorProfile = new TrapezoidProfile(
+                ArmConstants.ELEVATOR_CONSTRAINTS,
+                new TrapezoidProfile.State(targetElevatorPosition, 0),
+                new TrapezoidProfile.State(getElevatorMotorPositionRevolutions(), getElevatorMotorVelocityRevolutionsPerSecond())
+        );
+        lastElevatorMotorProfileGenerationTime = Timer.getFPGATimestamp();
+    }
+
+    private void setTargetAngleFromProfile() {
+        if (angleMotorProfile == null) {
+            stopAngleMotors();
+            return;
+        }
+        final TrapezoidProfile.State targetState = angleMotorProfile.calculate(getAngleMotorProfileTime());
+        masterAngleMotor.set(ArmConstants.ANGLE_PID_CONTROLLER.calculate(targetState.velocity));
+    }
+
+    private void setTargetPositionFromProfile() {
+        if (elevatorMotorProfile == null) {
+            stopElevatorMotors();
+            return;
+        }
+        final TrapezoidProfile.State targetState = elevatorMotorProfile.calculate(getElevatorMotorProfileTime());
+        masterElevatorMotor.set(ArmConstants.ELEVATOR_PID_CONTROLLER.calculate(targetState.velocity));
```

Comment:
You're on the right path. 
You used the `calculate` function wrong, didn't apply the power to both the master and the follower, and didn't set voltage.